src/test/ui/associated-types/hr-associated-type-bound-2.stderr

```diff
@@ -1,12 +1,10 @@
-error[E0599]: no method named `f` found for type `u32` in the current scope
+error[E0599]: the method `f` exists for type `u32`, but the following trait bounds were not satisfied:
+`<u32 as X<'b>>::U: Clone`
+which is required by `u32: X`
```

Comment:
Primary messages with newlines in them are not something that we've consciously tried to support, and I would argue we should avoid.